### ISSUE TITLE
[TypeScript]: The module “antlr4” or the associated type declarations were not found.ts(2307)

### ISSUE THREAD

**Issue by MarkusFaatz** [antlr/antlr4#4662]
After installing the package, TypeScript could not find the type definitions.

I only could get it to work by manually adding the antlr4's _src_ directory to my _tsconfig.json_ file like so

```
{
  "compilerOptions": {
    ...,
    "typeRoots": [
      "./node_modules/antlr4/src"
    ],
    ...
 }
}
```

I am not an expert in this but I guess that something is not working as expected with the types sections within the _package.json_ of the antlr npm package.

**Comment by ericvergnaud** [comment]
Presumably fixed by #4644

**Comment by MarkusFaatz** [comment]
Yes! I could verify the fix by applying the changes manually to the _package.json_ within the _node_modules/antlr4_ directory. 
Thanks a lot! 👍🏽
